pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -180,8 +180,9 @@ func (pl *CSILimits) filterAttachableVolumes(
 		pvc, err := pl.pvcLister.PersistentVolumeClaims(pod.Namespace).Get(pvcName)
 
 		if err != nil {
-			klog.V(5).InfoS("Unable to look up PVC info", "PVC", fmt.Sprintf("%s/%s", pod.Namespace, pvcName))
-			continue
+			// The PVC is required to proceed with Pod
```

Comment:
There's two scenarios we're concerned about: 1) The PVC is missing for the pod being scheduled 2) The PVC is missing for a different Pod that has already been scheduled. 

For the first, we want to fail, but for the 2nd we want to ignore so that a pod from another namespace cannot cause a different pod to fail scheduling.

Reply:
So what you are saying is that we have two pods, one which can run and one which can't because of a missing PVC, returning an error for the second pod will cause scheduling of *both* pods to fail? Why is returning an error not limited to just one pod?

Reply:
Perhaps it'll be clearer if you give an example?

Reply:
We discussed this in a meeting and @msau42 explained that this function gets called in a loop for new and existing pods. The decision at some point was to ignore existing pods with missing volumes.

I've changed the error case so that it only triggers for new pods.

@msau42: please take another look